.github/actions/build/action.yml

```diff
@@ -45,6 +45,7 @@ runs:
         NEXT_PUBLIC_TEST_GA_TRACKING_ID: ${{ fromJSON(inputs.secrets).NEXT_PUBLIC_TEST_GA_TRACKING_ID }}
         NEXT_PUBLIC_SAFE_APPS_GA_TRACKING_ID: ${{ fromJSON(inputs.secrets).NEXT_PUBLIC_SAFE_APPS_GA_TRACKING_ID }}
         NEXT_PUBLIC_SENTRY_DSN: ${{ fromJSON(inputs.secrets).NEXT_PUBLIC_SENTRY_DSN }}
+        NEXT_PUBLIC_DATADOG_CLIENT_TOKEN: ${{ fromJSON(inputs.secrets).NEXT_PUBLIC_DATADOG_CLIENT_TOKEN }}
```

Comment:
I think we also need to add this into the `.github/actions/build` file in order for it to work in the CI.